static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -134,13 +133,13 @@ const shouldShowEpic = (test: EpicABTest): boolean => {
 
     const tagsMatch = doTagsMatch(test);
 
-    return (
-        shouldShowReaderRevenue(test.showToContributorsAndSupporters) &&
-        onCompatiblePage &&
-        inCompatibleLocation &&
-        test.locationCheck(storedGeolocation) &&
-        tagsMatch
-    );
+    return !shouldHideReaderRevenue && test.showToContributorsAndSupporters
+        ? userIsSupporter()
+        : !userIsSupporter() &&
+              onCompatiblePage &&
+              inCompatibleLocation &&
+              test.locationCheck(storedGeolocation) &&
```

Comment:
I'd delete this `locationCheck` from everywhere it appears. It's not used currently and won't be in a world where locations are controlled just from the Google Sheet

Reply:
(it's particularly confusing having it next to `inCompatibleLocation`!)